app/namespaces/data_provider/data_provider_schema.py

```diff
@@ -10,7 +10,9 @@ class RecordsSchema(Schema):
     per_page = fields.Integer(allow_none=True)
     reverse = fields.Boolean(allow_none=True)
     research_fields = fields.Boolean(allow_none=True)
-    prioritize_estimates = fields.Boolean(allow_none=True)
+    estimates_subgroup = fields.String(validate=validate.OneOf(['all',
```

Comment:
v nit but would prefer the following naming, either:
['all', 'prioritized', 'primary'] or ['all_estimates', 'prioritized'_estimates, 'primary_estimates']

Reply:
sure